helm_deploy/offender-search/templates/secrets.yaml

```diff
@@ -4,4 +4,6 @@ metadata:
   name: {{ template "app.name" . }}
 type: Opaque
 data:
-  APPINSIGHTS_INSTRUMENTATIONKEY: {{ .Values.secrets.APPINSIGHTS_INSTRUMENTATIONKEY | b64enc | quote }}
\ No newline at end of file
+#  TODO: Fix app insights
+#  APPINSIGHTS_INSTRUMENTATIONKEY: {{ .Values.secrets.APPINSIGHTS_INSTRUMENTATIONKEY | b64enc | quote }}
+  APPINSIGHTS_INSTRUMENTATIONKEY: "Zm9vCg=="
```

Comment:
I think the issue is the secret is missing from AWS Secrets Manager - we can resolve that in a different PR if you liike

Reply:
actually, just checked it is in AWS secrets so not sure why that didn't work. maybe the circle job isn't correct

Reply:
have reverted this and is all working, not sure what the original problem was